mlflow/server/js/scripts/global-setup.js

```diff
@@ -5,7 +5,7 @@ module.exports = () => {
   // On windows, the timezone is not set with `TZ=GMT`. As a workaround, use `tzutil`.
   // This approach is taken from https://www.npmjs.com/package/set-tz.
   if (os.platform() === 'win32') {
-    const TZ = 'GMT Standard Time';
```

Comment:
`GMT Standard Time` represents the timezone containing Dublin, Edinburgh, Lisbon, **London** and adjust for DST.